lib/ddr/models/file_management.rb

```diff
@@ -20,7 +20,7 @@ module FileManagement
     #   Not required for external datastream classes
     #   or datastream instances having controlGroup 'E'.
     def add_file(file, dsid, mime_type: nil, external: false, original_filename: nil)
-      mime_type         ||= Ddr::Utils.mime_type_for(file)
+      mime_type         ||= MediaType.call(file) # XXX Should we use original_filename, if present?
```

Comment:
@dchandekstark My inclination is to say, yes, use original_filename, if present.  I guess a counter-argument is that there have been cases where it turned out that the "original filename" based on the user's submission had an incorrect extension -- i.e., it wasn't really the type of file the extension suggested it was.  I don't know if that type of scenario is applicable at this point or not.  @seanaery ?